app/assets/javascripts/field_filter.js.coffee

```diff
@@ -0,0 +1,17 @@
+class @FieldFilter
+  constructor: (@$fieldOrWrappers) ->
+
+  toggle: (showOrHide) ->
+    # https://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/
+    @$fieldOrWrappers.attr('aria-hidden', !showOrHide)
+
+    # disable inputs so they aren't submitted with the form
+    if @$fieldOrWrappers.is(':input')
```

Comment:
what about putting this in a well-named function rather than adding a comment about what it does!?